{{cookiecutter.github_repository}}/circle.yml

```diff
@@ -0,0 +1,87 @@
+## Customize the test machine
+machine:
+  python:
+    version: {{ '3.5' if cookiecutter.python3.lower() == 'y' else '2.7.4'}} # 2.7 will cause an error with py.test
+  environment:
+    # each command runs in its own shell, so just adding an export FOO=bar command won't work
+    DATABASE_URL: postgres://postgres@localhost/{{ cookiecutter.main_module }}
```

Comment:
How about having the `DATABASE_URL` use existing database. So we don't have to create one, making whole thing run faster. 
```
DATABASE_URL: "postgres://ubuntu@localhost/circle_test"
```